spec/spec.ddoc

```diff
@@ -5,6 +5,23 @@ DDOC_BLANKLINE = $(DIVC blankline)
 
 SUBNAV=$(SUBNAV_SPEC)
 SUBNAV_SPEC=
+<style type="text/css">
+body {
+  counter-reset: para h3;
+}
+p:before {
+  counter-increment: para;
+  content: counter(h3) "." counter(para) ": ";
+}
+h3:before {
+  counter-increment: h3;
+  content: "" counter(h3) ": ";
+  counter-reset: para;
+}
+h3 {
+  counter-reset: para;
+}
+</style>
```

Comment:
@CyberShadow the `style.css` is growing constantly in size, which makes adds a lot of unnecessary ballast to sites that want to use the dlang.org theme (I am not innocent about this). If we would use a modular CSS like SASS or LESS (or simply multiple files instead of a single `css/style.css`) it wouldn't be a big problem, but here I think that this is the right approach for CSS that only affects the spec pages.
However, `EXTRA_HEADERS` like it's used on the main `index.dd` are better suited for this:

```
EXTRA_HEADERS=$(T style,
...
)
```

https://github.com/dlang/dlang.org/blob/990d61d36a77fc62da1dd862f9e97013b8fd1346/index.dd#L554